### PR TITLE
Changed getBooks endpoint to return covers

### PR DESCRIPTION
**Pull request by bzhou93**
To test new getBooks, start up the backend server and use Postman to send a GET request to localhost:3000/users/getBooks.

The response may be hard to parse with just a glance, but the cover elements should have buffers if there is a cover to be retrieved from the server.  While it's annoying, you may want to look through the "Raw" response and count to make sure that all books are accounted for, and that those with covers have buffers representing the images.

As a side note, this code accommodates the slightly erroneous database table "book" by trimming whitespace.

### CODE REVIEW

**Comment by bzhou93** [comment]
Needs to be turned into an async function so that books can be loaded if a problem happens in the middle.

**Comment by gauravdesai** [comment]
Looks good! Great job!
